Update sickness.md (#977)
publiccodenet/about: activities/staff-information/sickness.md

The operations coordinator immediately reports the sick leave in [the portal of our absenteeism insurance](https://mijnwerkgeversportaal.acumen.nl/index3.html).
As soon as the employee is better it needs to be reported in the same portal. This can also be done retroactively if the operations coordinator is not present on the day the employee goes back to work.

### Long-term sick leave management

> In case an employee is sick for a longer time the ARBO doctor will be involved. Our ARBO service is [RegioPoortwachters](https://www.regiopoortwachters.nl/). A case manager from RegioPoortwachters will reach out to inquire about the sick employee within approx. 10 days after the initial sickness report in the insurance portal. The ARBO service portal is integrated with our insurance portal.

Reviewer: What is an ARBO doctor?

Reviewer 2: An Arbodienst doctor is a third party organizational doctor. _Arbo_ comes from the Dutch 'Arbeidsomstandighedenwet', shortened to 'arbowet' or employment situation law that tries to safeguard the employment situation of employees.

Reviewer: I suggest that this should be included (or linked to) in some way.

> In case an employee is sick for a longer time [an ARBO (company) doctor will be involved](https://www.arboned.nl/en/absence-support/company-doctor-as-specialist). Our ARBO service is [RegioPoortwachters](https://www.regiopoortwachters.nl/). A case manager from RegioPoortwachters will reach out to inquire about the sick employee within approx. 10 days after the initial sickness report in the insurance portal. The ARBO service portal is integrated with our insurance portal.

Reviewer 3: ```suggestion
In case an employee is sick for a longer time [an ARBO (company) doctor will be involved](https://www.arboned.nl/en/absence-support/company-doctor-as-specialist). Our ARBO service is [RegioPoortwachters](https://www.regiopoortwachters.nl/). A case manager from RegioPoortwachters will reach out to inquire about the sick employee within approximately 10 days after the initial sickness report in the insurance portal. The ARBO service portal is integrated with our insurance portal.
```

Reviewer 3: Separate question: who does RegioPoortwachters reach out to? The employee, or the Foundation?

> A meeting with an ARBO doctor will be planned to assess the situation of the employee. Depending on the outcome of the assessment, reintegration will be initialized when the situation allows. [A reintegration plan (plan van aanpak) will be drafted for employees who get ill](https://business.gov.nl/regulation/working-conditions-employees/). This plan should include the procedures all involved parties must follow. The director must approve the plan.

Reviewer 3: ```suggestion
The ARBO service will plan a meeting between the employee and an ARBO doctor to assess the employee's situation. Based on the outcome of the assessment, the employee will start reintegration when their situation allows. The ARBO service will draft a [reintegration plan (plan van aanpak)(https://business.gov.nl/regulation/working-conditions-employees/) for the employee. This plan should include the procedures all involved parties must follow. The director must approve the plan.
```

Reviewer 3: Is the director the chief executive, or a member of the board of directors?

### Reintegration management

> The case manager is the main contact person between the sick employee and the director. Updates are send regularly. It's important that the percentage of sickness vs reintegration hours are tracked in the insurance portal. E.g. if an employee is working 20% this means they are sick for 80%. This needs to be kept up-to-date as the insurance pay-out is based on the percentage of sickness.

Reviewer 3: Is the director the chief executive, or a member of the board of directors? (Update everywhere for clarity.)

Reviewer 3: Separately, is there any role for HR in this process?

Author: The director is the chief executive. Will review and update accordingly. Yes, it was my job to do but since Boris will take over these tasks and no replacement will be hired I wrote it as such to reflect the nearly current situation.

> The case manager is the main contact person between the sick employee and the director. Updates are send regularly. It's important that the percentage of sickness vs reintegration hours are tracked in the insurance portal. E.g. if an employee is working 20% this means they are sick for 80%. This needs to be kept up-to-date as the insurance pay-out is based on the percentage of sickness.

Reviewer 3: ```suggestion
The ARBO service case manager is the main contact person between the sick employee and the director, and will send updates regularly. It's important to track the percentage of sickness vs reintegration hours in the insurance portal. For example, if an employee is working 20%, this means they are sick for 80%. This needs to be kept up-to-date as the insurance reimbursement is based on the percentage of sickness.
```

> After a certain amount of time, approx. 1.5 months after the initial sickness report, the insurance company will request payslips of the sick employee to prepare for the insurance pay-out. The director will receive a mail about this. [The payslips are available for admins in Tentoo](https://about.publiccode.net/activities/tool-management/tentoo.html).

Reviewer 3: ```suggestion
Approximately 1.5 months after the initial sickness report, the insurance company will request the sick employee's payslips to prepare for the insurance payout. The director will receive an email about this. [Payslips are available to admins in Tentoo](https://about.publiccode.net/activities/tool-management/tentoo.html).
```

### UWV (Uitvoeringsinstituut Werknemersverzekeringen)

> If the employee has been sick for more than three quarters of a year, [it must be reported to the UWV](https://www.uwv.nl/werkgevers/werknemer-is-ziek/loondoorbetaling/werknemer-is-langdurig-ziek/index.aspx) no later than the first working day after the 42nd week of illness: the 42nd week report. Once that has been done we will receive a letter about the (first year) evaluation. During this evaluation the director and the employee discuss whether the approach has been successful. The ARBO case manager will be available for guidance and instructions when this moment comes.

Reviewer 3: ```suggestion
If the employee has been sick for more than three quarters of a year, [it must be reported to the UWV](https://www.uwv.nl/werkgevers/werknemer-is-ziek/loondoorbetaling/werknemer-is-langdurig-ziek/index.aspx). The 42nd week report must be submitted no later than the first working day after the 42nd week of illness. 

If the employee has not completed reintegration after a year, a one year evaluation must take place. During this evaluation the director and the employee discuss whether the integration approach has been successful, and a report is prepared. The ARBO case manager will be available for guidance and instructions.
```
